src/cli/commands/public-cmds/show-cmd.ts

```diff
@@ -14,12 +14,12 @@ export default class Show implements LegacyCommand {
   group: Group = 'info';
   alias = '';
   opts = [
-    ['j', 'json', 'return a json version of the component'],
-    ['r', 'remote', 'show a remote component'],
+    ['j', 'json', 'return component overview in json format'],
+    ['r', 'remote', 'show overview of a remote component'],
     ['v', 'versions', 'return a json of all the versions of the component'],
-    ['o', 'outdated', 'show latest version from the remote scope (if exists)'],
+    ['o', 'outdated', 'show latest version from the remote scope (if exists)'], // @david not sure how the flag name and description match?
```

Comment:
so in other words, it checks to see if the component is outdated?

Reply:
Yes, but it does it by showing an additional column of the version on the remote. (if I remember correctly it paints the version red in case it's outdated)

Reply:
ok, good to know - still think the description as is, is describing what they see, rather than the reason for using the command.
So i'm going to change to the following - lmk if you have comments:
"highlight outdated components, in comparison with their latest remote version"

Reply:
ok, that's good